packages/api/src/api/project/members/put.ts

```diff
@@ -0,0 +1,50 @@
+import { Request, Response } from 'express';
+import { DriverException, LockMode } from '@mikro-orm/core';
+import { Project, User } from '@hangar/database';
+import { Schema } from '@hangar/shared';
+import { logger } from '../../../utils/logger';
+import { validatePayload } from '../../../utils/validatePayload';
+
+export const put = async (req: Request, res: Response) => {
+  const { entityManager, user } = req;
+
+  const { errorHandled, data } = validatePayload({ req, res, schema: Schema.project.put });
+  if (errorHandled) return;
+
+  const project = await entityManager.findOneOrFail(Project, { id: data.projectId });
+  if (project.inviteCode !== data.inviteCode) {
+    res.status(400).send('Invalid invite code');
```

Comment:
Let's just reply ambiguously in this situation; we can also use a 403 here if you want to be consistent with the other endpoint that had this situation.
```suggestion
    res.sendStatus(404);
```